l10n-screenshots.sh

```diff
@@ -39,10 +39,11 @@ for lang in $LOCALES; do
     echo "$(date) Snapshotting $lang"
     mkdir "l10n-screenshots/$lang"
     fastlane snapshot --project Client.xcodeproj --scheme L10nSnapshotTests \
+        --number_of_retries 0 \
```

Comment:
@isabelrios We may still need retries in the tests. There may be some intermittent failures that I have not been able to investigate yet. I'm running `./l10n-screenshots.sh` with all available locales. There supposed to be 43 screenshots from each locales. For some locales, there are a few screenshots missing. I have not looked closely on which exact screenshots have been missing yet.

If the cron job does not have all screenshots for a locale, do we rerun the tests with the locale manually?

Reply:
My preliminary investigation says some tests other than `testLoginDetails()` could fail intermittently.